### ISSUE TITLE
Please consider Angular Material, lumX, Material UI

### ISSUE THREAD

**Issue by antitoxic** [Polymer/paper-elements#24]
Implementing Google material design is an awesome thing! I adore that fact that people are doing it.
I'm also addicted to web components. A round of loud applause :clap:  !

It puzzed me though that there's other implementations done on google material design. 

There is [Angular Material](https://github.com/angular/material). It's interesting for me because **both Anguarjs and Polymer are powerd by Google**.  Is it possible to **unify the two in some way**? At least making Polymer components and Anguar share the same animations and styling? Polymer components can use stylesheets, so if Anguar Material allows compilation of per-component styling and some selector name customisations they can both benefit, right?

2) [LumX](http://ui.lumapps.com/) is **another Angularjs implementation of material design**. In my research this looks the most complete implementations in terms of different components. I've cross-posed this to Anguar Material repo and I'm hoping that they merge.

**And the last thing** 3) - There's a facebook reactjs implementation of material design called [Material UI[(http://callemall.github.io/material-ui/). In my opinion it has the best dropdown animations and some other very carefully crafted details. Perhaps it could be of use to share some styling and animation, may be even contact the developers?

PS: there's also [Materialize CSS](http://materializecss.com/) which I'm using for extra components styling.

**Comment by marcoms** [comment]
First of all, I believe angular's and polymer's goals are different. Angular already has a huge amount of people using it, a large number probably for debian-grade stable projects. Polymer, on the other hand, has a little bit of experimental freedom wrt the features it can implement, the styles it uses, etc.

**Comment by antitoxic** [comment]
Yes, Polymer elements can make use of styling more freely as it's still in developer preview, however elements styling can be boosted with the work done by others on Angular, React and CSS implementations above.

Why not learn from each other? Currently I'm building a new project with Polymer and I am mixing SASS files from other frameworks to fill the gaps (styling for components like cards, lists, better dropdowns and so on.

It would be nice to see the good bits merged into paper elements.

**Comment by egranville** [comment]
+1 on this antitoxic,

Why don't they work together to create one Bugfree, Complete and Easier to debug framework that is updated quicker to accommodate new browsers. 

I'm also using Polymer to do a new project but find it still needs quite some work and I wouldn't say it's 100% production ready.
